package.json

```diff
@@ -15,26 +15,22 @@
   "bugs": {
     "url": "https://github.com/atlassian/react-beautiful-dnd/issues"
   },
-  "main": "lib/index.js",
-  "module": "esm/index.js",
+  "main": "dist/react-beautiful-dnd.cjs.js",
+  "module": "dist/react-beautiful-dnd.esm.js",
   "sideEffects": false,
   "files": [
-    "/lib",
     "/dist",
-    "/esm",
     "/src"
   ],
   "scripts": {
     "test": "jest --config ./jest.config.js && cross-env SNAPSHOT=check yarn build:dist",
     "validate": "yarn run lint && yarn run typecheck",
     "lint": "yarn eslint .",
     "typecheck": "flow check",
-    "build": "yarn run build:clean && yarn run build:lib && yarn run build:esm && yarn run build:flow && yarn run build:dist",
-    "build:clean": "rimraf lib && rimraf esm && rimraf dist",
-    "build:lib": "cross-env NODE_ENV=cjs babel src -d lib",
-    "build:esm": "babel src -d esm",
+    "build": "yarn run build:clean && yarn run build:dist && yarn run build:flow",
+    "build:clean": "rimraf dist",
     "build:dist": "rollup -c",
-    "build:flow": "echo \"// @flow\n\nexport * from '../src';\" | tee lib/index.js.flow esm/index.js.flow",
+    "build:flow": "echo \"// @flow\n\nexport * from '../src';\" > dist/react-beautiful-dnd.cjs.js.flow",
```

Comment:
Is flow okay with this? Do we want it also for the .esm.js file too?

Reply:
I don't think it's necessary. Users usually do not specify esm entry directly, but rely on bundler.